doc/howtos/booting-and-installation.rst

```diff
@@ -127,6 +127,58 @@ includes a checksum verification, you can also use the traditional :command:`dd`
 Unplug the removable media from your development system and you're ready to plug
 it into your target system.
 
+Installing to internal media on generic x86 EFI platforms
+=========================================================
+
+This section applies to those cases where the device is x86 compatible (ex: Gigabyte NUC).
+The simpler way is to follow these steps:
+
+#. Create a Bootable Media using an both an image that is compatible with the device
+   and a supported removable media (SD card or USB stick).
+#. Boot the device with the removable media created at the previous step.
+#. Once the device is booted, identify the booted media, for example by executing::
+
+      $ mount
```

Comment:
In a dev image, we're logged in as root so the the prompt is '#' (ok, I know I'm splitting hairs here...)